setup.py

```diff
@@ -53,11 +53,83 @@ def get_data_files():
     data_files = []
     ntrim = len(here + os.path.sep)
 
+    whitelist = (
+        'share/jupyterhub/static/favicon.ico',
+        'share/jupyterhub/static/components/bootstrap/LICENSE',
+        'share/jupyterhub/static/components/bootstrap/dist/js/bootstrap.min.js',
+        'share/jupyterhub/static/components/bootstrap/dist/js/npm.js',
+        'share/jupyterhub/static/components/bootstrap/dist/css/bootstrap.min.css',
+        'share/jupyterhub/static/components/bootstrap/dist/css/bootstrap-theme.min.css',
+        'share/jupyterhub/static/components/bootstrap/dist/css/bootstrap-theme.min.css.map',
+        'share/jupyterhub/static/components/bootstrap/dist/css/bootstrap.min.css.map',
+        'share/jupyterhub/static/components/bootstrap/dist/fonts/glyphicons-halflings-regular.eot',
+        'share/jupyterhub/static/components/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2',
+        'share/jupyterhub/static/components/bootstrap/dist/fonts/glyphicons-halflings-regular.ttf',
+        'share/jupyterhub/static/components/bootstrap/dist/fonts/glyphicons-halflings-regular.svg',
+        'share/jupyterhub/static/components/bootstrap/dist/fonts/glyphicons-halflings-regular.woff',
+        'share/jupyterhub/static/components/jquery/LICENSE.txt',
+        'share/jupyterhub/static/components/jquery/external/sizzle/LICENSE.txt',
+        'share/jupyterhub/static/components/jquery/external/sizzle/dist/sizzle.min.js',
+        'share/jupyterhub/static/components/jquery/external/sizzle/dist/sizzle.min.map',
+        'share/jupyterhub/static/components/jquery/dist/core.js',
+        'share/jupyterhub/static/components/jquery/dist/jquery.slim.min.js',
+        'share/jupyterhub/static/components/jquery/dist/jquery.slim.min.map',
+        'share/jupyterhub/static/components/jquery/dist/jquery.min.js',
+        'share/jupyterhub/static/components/jquery/dist/jquery.min.map',
+        'share/jupyterhub/static/components/moment/moment.js',
+        'share/jupyterhub/static/components/moment/LICENSE',
+        'share/jupyterhub/static/components/moment/ender.js',
+        'share/jupyterhub/static/components/moment/min/moment.min.js',
+        'share/jupyterhub/static/components/moment/min/locales.min.js',
+        'share/jupyterhub/static/components/moment/min/moment-with-locales.min.js',
+        'share/jupyterhub/static/components/font-awesome/css/font-awesome.css.map',
+        'share/jupyterhub/static/components/font-awesome/css/font-awesome.css',
+        'share/jupyterhub/static/components/font-awesome/css/font-awesome.min.css',
```

Comment:
Should there be a `font-awesome.min.css.map`?

Reply:
No, I don't think so